src/mmw/apps/modeling/serializers.py

```diff
@@ -15,10 +17,21 @@ class Meta:
         fields = ('id', 'username', 'email')
 
 
+class ModificationsField(serializers.BaseSerializer):
+
+    def to_representation(self, obj):
+        return obj
```

Comment:
I think we should be returning a json object here, so it's deserialized appropriately for the client.  Consider returning `json.loads(obj)` here instead and making sure the client is not specifically expecting a string.